vpr/src/base/ShowSetup.cpp

```diff
@@ -657,6 +657,7 @@ static void ShowAnalyticalPlacerOpts(const t_ap_opts& APOpts) {
 
     VTR_LOG("AnalyticalPlacerOpts.ap_timing_tradeoff: %f\n", APOpts.ap_timing_tradeoff);
     VTR_LOG("AnalyticalPlacerOpts.log_verbosity: %d\n", APOpts.log_verbosity);
+    VTR_LOG("AnalyticalPlacerOpts.generate_mass_report: %s\n", APOpts.generate_mass_report ? "true" : "false");
```

Comment:
I'm not sure what the convention is for displaying parameter values in the vpr.out report. I think it would be better to avoid printing all parameter values and instead only include those that are necessary for analyzing the results. If you agree, I suggest we omit the value of the generate_mass_report parameter (or any similar parameter that simply controls whether a report should be generated).